docs/guide/roles-and-multi-sig.md

```diff
@@ -0,0 +1,261 @@
+
+```
+{
+  "type": "sigs/multi",
+  "data": {
+    "tx": {
+      "type": "chain/tx",
+      "data": {
+        "chain_id": "test_chain_id",
+        "expires_at": 0,
+        "tx": {
+          "type": "nonce",
+          "data": {
+            "sequence": 1,
+            "signers": [
+              {
+                "chain": "",
+                "app": "sigs",
+                "addr": "65D406E028319289A0706E294F3B764F44EBA3CF"
+              }
+            ],
+            "tx": {
+              "type": "role/assume",
+              "data": {
+                "role": "10CAFE4E",
+                "tx": {
+                  "type": "coin/send",
+                  "data": {
+                    "inputs": [
+                      {
+                        "address": {
+                          "chain": "",
+                          "app": "role",
+                          "addr": "10CAFE4E"
+                        },
+                        "coins": [
+                          {
+                            "denom": "mycoin",
+                            "amount": 6000
+                          }
+                        ]
+                      }
+                    ],
+                    "outputs": [
+                      {
+                        "address": {
+                          "chain": "",
+                          "app": "sigs",
+                          "addr": "65D406E028319289A0706E294F3B764F44EBA3CF"
+                        },
+                        "coins": [
+                          {
+                            "denom": "mycoin",
+                            "amount": 6000
+                          }
+                        ]
+                      }
+                    ]
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "signatures": [
+      {
+        "Sig": {
+          "type": "ed25519",
+          "data": "A38F73BF2D109015E4B0B6782C84875292D5FAA75F0E3362C9BD29B16CB15D57FDF0553205E7A33C740319397A434B7C31CBB10BE7F8270C9984C5567D2DC002"
+        },
+        "Pubkey": {
+          "type": "ed25519",
+          "data": "6ED38C7453148DD90DFC41D9339CE45BEFA5EB505FD7E93D85E71DFFDAFD9B8F"
+        }
+      }
+    ]
+  }
+}
+```
+
+and it is loaded by the next command.
+
+With the transaction prepared, but not sent, we'll have `igor` sign and send the prepared transaction:
+
+```
+basecli tx --in=tx.json --name=igor
+```
+
+which will give output similar to:
+
+```
+{
+  "check_tx": {
+    "code": 0,
+    "data": "",
+    "log": ""
+  },
+  "deliver_tx": {
+    "code": 0,
+    "data": "",
+    "log": ""
+  },
+  "hash": "E345BDDED9517EB2CAAF5E30AFF3AB38A1172833",
+  "height": 2673
+}
+```
+
+and voila! That's the basics for creating roles and sending multi-sig transactions. For 3 of 3, you'd repeat the last step for rich as well (recall that poor initiated the multi-sig transaction). We can check the balance of the role:
```

Comment:
Actually the intermediate state would look like:

`basecli tx --in=tx.json --name=igor --prepare=tx2.json`

The prepare means to output it to disk rather than send it.